week-03-ajax-json-express/day-02-ajax/dusk-get-post/solutions.md

```diff
@@ -0,0 +1,16 @@
+## Solutions - AJAX: GET & POST
+
+
+#### Postman
+
+1. Use Postman to `GET` all the `books`. Now get just one book using the `_id`.
+
+  ![screen shot 2015-11-05 at 18 02 06](https://cloud.githubusercontent.com/assets/7833470/10987367/84e1849a-83e7-11e5-862b-5d03d0b119f1.png)
+
+2. Use Postman to send a `POST` request to create a new book.
+
+  ![screen shot 2015-11-05 at 18 03 04](https://cloud.githubusercontent.com/assets/7833470/10987366/84cefd3e-83e7-11e5-85be-19832324f1ca.png)
+
```

Comment:
Are you planning to add solutions for 3,4,5?
How would you request your book, using your id?
How would you update your book to change the date to your birthdate?
How would you delete your book?